server/server.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const compression = require('compression');
+const controllers = require('./controllers/routes');
+const app = express();
+
+const mongoose = require('mongoose');
+var cors = require('cors');
+const bodyParser = require('body-parser');
+const logger = require('morgan');
+const Data = require('./data');
+
+const API_PORT = 3001;
+
+app.use(cors());
```

Comment:
Is there specific reason why are you using Cross-Origin Resource Sharing package?